packages/synapse-react-client/test/lib/containers/personal_access_token/CreateAccessTokenModal.test.tsx

```diff
@@ -25,6 +25,23 @@ function renderComponent(props: CreateAccessTokenModalProps) {
   })
 }
 
+function setUp(props: CreateAccessTokenModalProps) {
+  const user = userEvent.setup()
```

Comment:
react-testing-library [docs](https://testing-library.com/docs/user-event/intro#writing-tests-with-userevent) recommend invoking `userEvent.setup` before component is rendered

Reply:
Noted!